apps/publish/enqueue/enqueue_service.py

```diff
@@ -345,6 +345,13 @@ def publish_package(self, package, target_subscribers):
             if temp_queued:
                 queued = temp_queued
 
+            delivery_types = [d['delivery_type'] for d in self.get_destinations(subscriber)]
+            is_content_api_delivery = 'content_api' in delivery_types
```

Comment:
afaik there is no content_api delivery type

Reply:
@petrjasek it's used in several places (e.g. [there](https://github.com/superdesk/superdesk-core/blob/master/superdesk/publish/publish_queue.py#L146) and [there](https://github.com/superdesk/superdesk-core/blob/master/apps/publish/enqueue/enqueue_service.py#L499)

It is set [there](https://github.com/superdesk/superdesk-core/blob/master/apps/publish/enqueue/enqueue_service.py#L353) and the code is working anyway.

Reply:
I see, thx